.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -51,3 +51,7 @@ If yes, please enter a release note below:
 ```release-note
 
 ```
+
+**PR Checklist**
+
+- [ ] the `CHANGELOG.md` was updated with any user-facing changes
```

Comment:
If we're doing this, we probably want to remove the release-note section above

Reply:
:+1: Changed in 1e06e53e677c12d76f8fe9ec6570ffea61908737